py/selenium/webdriver/common/action_chains.py

```diff
@@ -21,6 +21,9 @@
 from selenium.webdriver.remote.webelement import WebElement
```

Comment:
We have quite a lot of typing tech debt, could we type this new code?

Reply:
@symonk I don't understand the term _'typing tech debt'_ Can you please tell me what it means? Also, I don't think I changed that code. Let me know if it still needs to be corrected after the latest commit.